helmexec/exec_test.go

```diff
@@ -258,7 +286,7 @@ func Test_TestRelease_Flags(t *testing.T) {
 	var buffer bytes.Buffer
 	logger := NewLogger(&buffer, "debug")
 	helm := MockExecer(logger, "dev")
-	helm.TestRelease("release", "--cleanup", "--timeout", "60")
+	helm.TestRelease(HelmContext{Tillerless: false}, "release", "--cleanup", "--timeout", "60")
```

Comment:
Nit but can we safely omit Tillerless:false as its the default(zero) value?

Reply:
What do you mean by "nit"?
OK, thanks. I never did write any go until previous PR... so I'm learning...  ;-)

Reply:
Thanks for taking your time :)

"Nit" stands for "nitpick" so I mean "I'm hesitant to point out such a small thing even though your PR is awesome overall, but…" :relaxed: